torchgen/model.py

```diff
@@ -2542,7 +2577,10 @@ def root_name(self) -> str:
     def composite(self) -> bool:
         # We currently assert that the "group" is consistent.
         # If the view op is composite, then its view_inplace op is too.
-        return self.view.has_composite_implicit_autograd_kernel
+        return (
+            self.view.has_composite_implicit_autograd_kernel
+            or self.view.has_composite_implicit_autograd_nested_tensor_kernel
```

Comment:
same comment as https://github.com/pytorch/pytorch/pull/82754/files#r950522386 (I don't think we need this)